core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -163,6 +164,13 @@
       "Leave blank to use Jetty's default.";
   protected static final String SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_DEFAULT = "";
 
+  public static final String AUTHENTICATION_METHOD = "authentication.method";
```

Comment:
Can you append `_CONFIG` to each of these config options to have consistent naming with other configuration parameters?